spec/acceptance/nodesets/ubuntu-16-docker.yml

```diff
@@ -0,0 +1,12 @@
+HOSTS:
+  ubuntu-16-docker:
+    roles:
+      - master
+    platform: ubuntu-1404-x86_64
```

Comment:
If I change this to 14.04, I see most things work but tests eventually fail with:

    /home/daniel/puppet-opendaylight/spec/spec_helper_acceptance.rb:195:in `generic_validations': Unexpected RS_SET (host OS):  (RuntimeError)